web/libs/editor/src/components/Timeline/Types.ts

```diff
@@ -73,17 +77,22 @@ export interface TimelineViewProps {
   onAddRegion?: TimelineProps["onAddRegion"];
   onDeleteRegion?: TimelineProps["onDeleteRegion"];
   onSelectRegion?: TimelineProps["onSelectRegion"];
+  onStartDrawing?: TimelineProps["onStartDrawing"];
+  onFinishDrawing?: TimelineProps["onFinishDrawing"];
   onVolumeChange?: TimelineProps["onVolumeChange"];
   onSpeedChange?: TimelineProps["onSpeedChange"];
 }
 
 export interface TimelineRegion {
   id: string;
+  index?: number;
   label: string;
   color: string;
   visible: boolean;
   selected: boolean;
   sequence: TimelineRegionKeyframe[];
+  /** is this timeline region with spans */
+  timeline?: boolean;
```

Comment:
Shouldn't it be something like `isTimeline`?

Reply:
I'm following the style of other props here (`selected`, `visible`) and I think that for small internal types it's okay